src/components/Button/Button.js

```diff
@@ -44,20 +44,15 @@ export const Button = styled.button`
       transform: scale(1);
     }
   }
+`;
+
+export const NakedButton = styled(Button)`
+  background: none;
+  border: none;
+  display: flex;
+  flex-direction: column;
 
-  svg {
-    ${utils.padding};
-    fill: ${props => (props.fill ? props.theme.textPrimary : 'transparent')};
-    height: ${props => (props.iconSize ? `${props.iconSize}px` : '18px')};
-    margin-right: ${props => (props.stacked ? '0' : '6px')};
-    margin-bottom: ${props => (props.stacked ? '6px' : '0')};
-    stroke: ${props =>
-      props.fill ? 'transparent' : `${props.theme.textPrimary}`};
-    stroke: ${props => props.stacked && `${props.theme.textPrimary}`};
-    stroke-width: ${props =>
-      props.strokeWidth ? `${props.strokeWidth}px` : '6px'};
-    stroke-width: ${props => props.fill && '6px'};
-    transition: 400ms ${EASE_OUT_EXPO};
-    width: ${props => (props.iconSize ? `${props.iconSize}px` : '18px')};
+  p {
```

Comment:
I am not sure if its semantic to have a `p` inside of a `button`. Also `p` is block while `button` is inline so we might get unexpected behavior. https://www.w3.org/TR/html5/sec-forms.html#elementdef-button

Could we achieve this with padding and making the parent `inline-block`?

Reply:
changed to `<span>` instead, that is valid